changelog/fragments/1672911992-v2-control-protocol.yaml

```
@@ -0,0 +1,32 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: breaking-change
+
+# Change summary; a 80ish characters long description of the change.
+summary: |
+  New control protocol for the Elastic Agent. Allows among other things to simplify new inputs development.
```

```suggestion
  New control protocol between the Elastic Agent and its subprocesses. Enables per integration health reporting and simplifies new input development, among other things.
```

Let's add more detail to this. I think we should specifically mention that it allows per integration health reporting.